.github/workflows/benchmark.main.kts

```diff
@@ -79,11 +80,10 @@ workflow(
     ) {
         uses(action = Checkout())
         uses(
-            name = "Set up JDK",
+            name = "Set up Gradle Daemon JDK",
             action = SetupJava(
-                javaVersion = "11",
-                distribution = SetupJava.Distribution.Zulu,
-                cache = SetupJava.BuildPlatform.Gradle,
+                javaVersion = "21",
+                distribution = Temurin,
```

Comment:
nit. Out of curiosity, why change Zulu to Temurin? I'm generally not familiar with each vendor's characteristics, so I'm trying to learn something here, and also learn a justification for changing it.

Reply:
I hope @aSemy can answer this

Reply:
There's no strong reason, I generally try to use Temurin wherever possible.

There's a flowchart here https://stackoverflow.com/questions/61335109/what-is-the-difference-between-azul-openjdk-zulu-openjdk-and-openjdk and Temurin is the first.

It makes zero practical difference.

Reply:
Do you mean https://i.sstatic.net/P4Gmt.png? I don't see Temurin there :sweat_smile: I suggest reverting this change from this PR since it's unrelated, and we can move it to a separate PR.

Reply:
Oh, right, Temurin is also known as 'Eclipse Adoptium'

https://adoptium.net/en-GB/docs/faq/

Reply:
Got it, ok! Let's make this change, just in a separate PR. I'll handle this PR.

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/259